modules/ossm-auto-route-1x.adoc

```diff
@@ -36,7 +36,7 @@ spec:
 [id="ossm-auto-route-subdomains_{context}"]
 == Subdomains
 
-{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example `*.domain.com`, are supported but not by default.
+{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example `*.domain.com`, are supported but not by default. Configure an {product-title} wildcard policy before configuring a wildcard host Gateway. For more information, see the Links section.
```

Comment:
```suggestion
{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example `*.domain.com`, are supported but not by default. Configure an {product-title} wildcard policy before configuring a wildcard host Gateway. For more information, see the "Links" section.
```

Reply:
IBM SG - Use double quotation marks with some citations that are not links, such as:

- Chapter titles and section titles
- Entries within a blog
- References to industry standards
- Titles of webcasts, web conferences, and
- online broadcasts
- Topic titles in IBM Knowledge Center or in
- an informa

tion center